.devcontainer/Dockerfile

```diff
@@ -2,5 +2,14 @@ FROM ghcr.io/containerbase/node:18.16.0@sha256:fc48ac97eb5d0dee207d4a336a52d7d3b
 
 USER root
 
+RUN ls
+
+RUN apt update; \
+    apt install -y make g++
+
+# renovate: datasource=github-releases
+RUN install-tool python 3.11.3
+
 # renovate: datasource=npm
 RUN install-tool yarn 1.22.19
+RUN yarn install
```

Comment:
are you sure the local repo is mounted while building the image?

Reply:
I am checking with the devcontainers community whether it is expected that the local repo to be mounted while building the image.  devcontainers/discussions#44